devel-tools/vignetting/imsim-vignetting.yaml

```diff
@@ -0,0 +1,33 @@
+# Use baseline configuration
+modules: [imsim]
+template: imsim-config
```

Comment:
I think let's copy the config file and edit it, rather than use a template.  I'm concerned that we'll make various changes to the base imsim-config over time, and this will become broken.